test/integration/narrow/routes/collaboration.test.js

```diff
@@ -1,6 +1,48 @@
-const { getCookieHeader, getCrumbCookie, crumbToken } = require('./test-helper')
+const { crumbToken } = require('./test-helper')
 describe('Collaboration page', () => {
-  let crumCookie
+  const project = ['some fake project']
+  const irrigatedCrops = 'some fake crop'
+  const irrigatedLandCurrent = '123'
+  const irrigatedLandTarget = '456'
+  const waterSourceCurrent = ['some source 1']
+  const waterSourcePlanned = ['some source 2', 'another source']
+  const irrigationCurrent = ['some source 2', 'another source']
+  const irrigationPlanned = ['some souce 2']
+  const productivity = ['some option-1', 'some option-2']
+  const collaboration = 'some fake data'
+
+  jest.mock('../../../../app/helpers/session', () => ({
```

Comment:
Can we make the code simpler in the mock cases for test files like this:
`const SOME_LIST = {
	irrigatedCrops: 'some fake crop',
	waterSourceCurrent: ['some source 1'],
	waterSourcePlanned: ['some source 2', 'another source'],
	...
}
  
jest.mock('../../../../app/helpers/session', () => ({
	setYarValue: (request, key, value) => null,
	getYarValue: (request, key) => {

  if (Object.keys(SOME_LIST).includes(key)) { return SOME_LIST[key] }
		else { return 'Error'}
	}
}))`

Reply:
thanks Ogbu for the code yes i agree but as i have to change 30 tests individually I will create a ticket for the tech debt for this one.

Reply:
ok